### PR TITLE
Annotations, minmax, study

### LINKED ISSUE #5: Update study to reference live publication
The publication is now available live at https://doi.org/10.1126/science.aar7462 (PMID: 30002254)

### PR DESCRIPTION
**Pull request by dominikl**
This PR adds some manual corrections to the annotations.csv, a script to calculate and set the minmax values for the rendering settings and the study.txt.

To do: Study type and experiment description are still missing from the study.txt.

Fixes #5

### CODE REVIEW

File: idr0045-study.txt [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
...nm xy and 520nm z resolution using 488, 561 and 642 laser lines.	SupplementaryFigure09 - Zygotes were isolated from C57BL/6J x C3H/He F1 females mated with C57BL/6J x C3H/He F1 males and fixed and stained as described (https://doi.org/10.1101/198275). Embryos were imaged on a SP8 Leica confocal microscope equipped with a 63* C-Apochromat 1.2 NA water immersion objective lens. Images of embryos were acquired at 90 nm in xy and 360 nm in z.	SupplementaryFigure10A - Zygotes were isolated from C57BL/6J x C3H/He F1 females mated with C57BL/6J x C3H/He F1 males and fixed and stained as described (https://doi.org/10.1101/198275). Embryos were imaged on a SP8 Leica confocal microscope equipped with a 63* C-Apochromat 1.2 NA water immersion objective lens. Images of embryos were acquired at 90 nm in xy and 360 nm in z.	SupplementaryFigure10B - Zygotes were isolated from C57BL/6J x C3H/He F1 females mated with C57BL/6J x C3H/He F1 males and fixed and stained as described (https://doi.org/10.1101/198275). Embryos were imaged on a SP8 Leica confocal microscope equipped with a 63* C-Apochromat 1.2 NA water immersion objective lens. Images of embryos were acquired at 90 nm in xy and 360 nm in z.
```
Comment (sbesson) [inline idr0045-study.txt]: Only the first part is put as the project description. The rest should probably be moved to `Experiment Comments`

File: idr0045-study.txt [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+Experiment Example Images		Embryo3	MicrotubuleBallStage4, Prometaphase1, Prometaphase3, Metaphase4	PM10, PM15	Embryo05						Zygote/Cell03, 2Cell/2-Cell-10-1, 4Cell/4-Cell-11-1 and 8Cell/8-Cell-14-1		NocMo/Nocodazole/Embryo2, NocMo/Monastrol/Embryo3, NocMo/Washout/Embryo1, MoNoc/Monastrol/Embryo1, MoNoc/NocodazoleEmbryo1 and MoNoc/Washout/Embryo1	Untreated/Zygote/08, Untreated/2-Cell/20, Untreated/8-Cell/02, NocMo/2-Cell/37, NocMo/8-Cell/12, MoNoc/2-Cell/22 and MoNoc/8-Cell/12	Untreated/Zygote/15, Untreated/2-Cell/06, Untreated/8-Cell/03, NocMo/2-Cell/01, NocMo/8-Cell/05, MoNoc/2-Cell/05 and MoNoc/8-Cell/02	Untreated/Zygote/04, Untreated/2-Cell/18, Untreated/8-Cell/01, NocMo/2-Cell/05, NocMo/8-Cell/11, MoNoc/2-Cel/12 and MoNoc/8-Cell/11
```
Comment (sbesson) [inline idr0045-study.txt]: I will take a look and find the URLs of the representative images

File: idr0045-study.txt [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+Experiment Imaging Method	light sheet microscopy; confocal microscopy	light sheet microscopy	confocal microscopy	confocal microscopy	light sheet microscopy	light sheet microscopy	light sheet microscopy	light sheet microscopy	confocal microscopy	light sheet microscopy	light sheet microscopy	light sheet microscopy	light sheet microscopy	confocal microscopy	confocal microscopy	confocal microscopy
```
Comment (sbesson) [inline idr0045-study.txt]: This field and the one below should probably be reduced to 2 columns only: `light sheet microscopy` and `confocal microscopy` and the map annotation should have only 2 entries

Comment (dominikl) [reply idr0045-study.txt]: The problem is that the "Experiments" (Datasets) have different methods, some are light sheet, some are confocoal. So the second column is a summary which describes the "Experiment" (in the IDR sense) ('light sheet microscopy; confocal microscopy') and the other columns are specific to the dataset. I could reduce it to two columns but then loose the context of which dataset is which method.

Comment (dominikl) [reply idr0045-study.txt]: Well, actuall this information is also still available in the description. Ok. I'll reduce the number of columns, and also put the "Experiment" (Dataset) descriptions into the "Experiment Comments" field.

File: idr0045-study.txt (hunk 2 above)
Comment (dominikl) [reply idr0045-study.txt]: Shall I just remove them? Seems to be a bit many example images.

Comment (sbesson) [reply idr0045-study.txt]: yes

File: idr0045-study.txt [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+Study Person First Name	jan.ellenberg@embl.de			
+Study Person Email	EMBL Heidelberg, Meyerhofstrasse 1, 69117 Heidelberg, Germany EMBL Heidelberg, Meyerhofstrasse 1, 69117 Heidelberg, Germany			
+Study Person Address	principal investigator	submitter		
+Study Person Roles				
+				
+# Study License and Data DOI	CC-BY 4.0			
+Study License	https://creativecommons.org/licenses/by/4.0/			
+Study License URL	University of Dundee			
+Study Data Publisher	https://doi.org/10.17867/10000115			
+Study Data DOI
```
Comment (sbesson) [inline idr0045-study.txt]: Looks like the values and the keys were shifted by one row. Also  add `Study Copyright\tReichman et al`?

**Pull request merged**